common/src/main/java/tachyon/security/PlainSaslServer.java

```diff
@@ -39,7 +38,7 @@
  * 3.Write a JCA provider that registers the factory
  */
 public class PlainSaslServer implements SaslServer {
-  private String mAuthcid;
+  private String mAuthzid;
```

Comment:
I think the username is authorization ID, which is mAuthcid. Why do we want to change it here? @shenguoquan Can you also review this? Thanks!